src/lib/components/product.svelte

```diff
@@ -5,42 +5,98 @@
   export let image: string
   export let name: string
   export let brand: string
+  export let size: string | undefined
+  export let region: string | undefined
 </script>
 
 <!-- <a href="#"> -->
-<div in:fade={{ duration: 300, delay: index * 150 }} class="product">
+<div in:fade={{ duration: 300, delay: index * 150 }} class="card">
   <img src={image} alt="" />
-  <h2>{brand}</h2>
-  <h4>{name}</h4>
+
+  <div class="content">
+    <h2 class="title">{brand} {name}</h2>
```

Comment:
Spaces added in html are not safe, we have been over this a number of times already. you need to string template og escape the space

Reply:
still an issue